### PR TITLE
Feature/aoaf 503 welsh for page

### PR DESCRIPTION
**Pull request by wdhda-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by ch-skathirvel — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/handlers/confirm_company/index.ts [hunk 1]
```diff
@@ -113,5 +115,10 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
         }
         setPresenterAccountDetails(req, { ...presenterAccountDetails, companyName });
     }
+
+    private appendCompanyNumberQuery(url: String, req: Request) {
+        const companyNumber = this.getCompanyNumberFromRequest(req);
+        return url + "?" + QueryParameters.COMPANY_NUMBER + "=" + companyNumber;
```
Comment (ch-skathirvel) [inline src/routers/handlers/confirm_company/index.ts:121]: Instead concat with +, can we use like below?
return `${url}?${QueryParameters.COMPANY_NUMBER}=companyNumber`;

**Review by ch-skathirvel — CHANGES_REQUESTED** [review]

**Review by ch-skathirvel — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**